src/current/cockroachcloud/provision-a-cluster-with-terraform.md

```diff
@@ -179,6 +188,78 @@ The `terraform show` command shows detailed information of your cluster resource
 terraform show
 ~~~
 
+## Change a cluster's plan
+
+To change a CockroachDB {{ site.data.products.basic }} cluster's plan to CockroachDB {{ site.data.products.standard }} in place, or vice versa, you can use Terraform or the CockroachDB {{ site.data.products.cloud }} API.
+
+<div class="filters clearfix">
+    <button class="filter-button page-level" data-scope="terraform">Terraform</button>
+    <button class="filter-button page-level" data-scope="api"><strong>CockroachDB Cloud API</button>
+</div>
+
+{{site.data.alerts.callout_info}}
+To migrate between CockroachDB {{ site.data.products.advanced }} and CockroachDB {{ site.data.products.standard }}, you must create and configure a new cluster, back up the existing cluster's data, and restore the backup to the new cluster. Migration in place is not supported. Refer to [Customer-owned backups]({% link cockroachcloud/backup-and-restore-overview.md %}#customer-owned-backups).
+{{site.data.alerts.end}}
+
+<section class="filter-content" markdown="1" data-scope="terraform">
+
+To migrate from CockroachDB {{ site.data.products.basic }} to CockroachDB {{ site.data.products.standard }} in place:
+
+1. Edit the cluster's Terraform template:
+    - Change `plan` to `STANDARD`.
+    - Replace the contents of `serverless {}` (which may be empty) with the provisioned vCPUs for the cluster. This field is required for CockroachDB {{ site.data.products.standard }}. It is not possible to set storage limitations on CockroachDB {{ site.data.products.standard }}.
+      {% include_cached copy-clipboard.html %}
+      ~~~ hcl
+            serverless = {
```

Comment:
```suggestion
        serverless = {
```
Removing four spaces before serverless for proper alignment

Reply:
Done